spec/api/documentation/organizations_api_spec.rb

```diff
@@ -90,6 +91,48 @@
       end
     end
 
+    describe 'Isolation Segments (experimental)' do
+      let(:isolation_segment_model) { VCAP::CloudController::IsolationSegmentModel.make }
+
+      before do
+        organization.add_manager(everything_user)
+        isolation_segment_model.add_organization(organization)
+      end
+
+      get '/v2/organizations/:guid/isolation_segments (experimental)' do
+        example 'get the associated Isolation Segments for the Organization' do
+          client.get "v2/organizations/#{organization.guid}/isolation_segments", {}, headers
+          expect(status).to eq(200)
+
+          expected_response = {
+            'pagination' => {
+              'total_results' =>  1,
+              'total_pages'   =>  1,
+              'first'         =>  { 'href' => "/v2/organizations/#{organization.guid}/isolation_segments?page=1&per_page=50" },
+              'last'          =>  { 'href' => "/v2/organizations/#{organization.guid}/isolation_segments?page=1&per_page=50" },
+              'next'          =>  nil,
+              'previous'      =>  nil
+            },
+            'resources' => [
+              {
+                'name'        => isolation_segment_model.name,
+                'guid'        => isolation_segment_model.guid,
+                'created_at'  => iso8601,
+                'updated_at'  => nil,
+                'links'       => {
```

Comment:
@Gerg @ScarletTanager I remember that we considered the body the same, but we never expected to see links or v3 pagination on the v2 api